packages/discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -617,24 +610,17 @@ def validate_access_conditions(params: ManageEntityParameters):
             raise IndexingValidationError(
                 f"Track {params.entity_id} has an invalid number of stream conditions"
             )
-        if is_downloadable:
-            # if stream gated and downloadable, must be download gated
-            if not is_download_gated:
-                raise IndexingValidationError(
-                    f"Track {params.entity_id} is stream gated but not download gated"
-                )
-            # if stream gated and downloadable, stream conditions must be same as download conditions
-            if stream_conditions != download_conditions:
-                raise IndexingValidationError(
-                    f"Track {params.entity_id} stream conditions do not match download conditions"
-                )
-
-    if is_download_gated:
-        # if download gated, must be downloadable
-        if not is_downloadable:
+        # if stream gated, must be download gated
+        if not is_download_gated:
```

Comment:
just double-checking, if track is stream-gated but has no downloadable assets, we still make download_conditions inherit from stream_conditions and set is_download_gated?